src/ctia/entity/indicator.clj

```diff
@@ -161,7 +161,29 @@
     :external-id-capabilities :read-indicator
     :can-aggregate?           true
     :histogram-fields         indicator-histogram-fields
-    :enumerable-fields        indicator-enumerable-fields}))
+    :enumerable-fields        indicator-enumerable-fields
+    :searchable-fields        #{:id
+                                :composite_indicator_expression.indicator_ids
+                                :composite_indicator_expression.operator
+                                :confidence
+                                :description
+                                :external_ids
+                                :external_references.description
+                                :external_references.external_id
+                                :external_references.hashes
+                                :external_references.source_name
+                                :external_references.url
+                                :indicator_type
+                                :kill_chain_phases.kill_chain_name
+                                :kill_chain_phases.phase_name
+                                :likely_impact
+                                :producer
+                                :severity
+                                :short_description
+                                :specification
+                                :tags
+                                :test_mechanisms
+                                :title}}))
```

Comment:
```suggestion
    :searchable-fields        #{:id
                                :source
                                :confidence
                                :description
                                :indicator_type
                                :kill_chain_phases.kill_chain_name
                                :kill_chain_phases.phase_name
                                :likely_impact
                                :producer
                                :severity
                                :short_description
                                :tags
                                :test_mechanisms
                                :title}}))
```

this is an example of specification https://github.com/threatgrid/ctim/blob/f6ffae8bbc986ba9eed13d7d4fdf1d6f1519f552/src/ctim/examples/indicators.cljc#L42
you cannot add it as a whole
```
   :specification {:type "Judgement"
                   :judgements ["http://example.com/ctia/judgement/judgement-1000efcc-3474-498d-a655-0e0dc57f84ba"]
                   :required_judgements [{:judgement_id "http://example.com/ctia/judgement/judgement-1000efcc-3474-498d-a655-0e0dc57f84ba"
                                          :source "source"
                                          :relationship "relationship"
                                          :confidence "High"}]}